docs/docs-content/enterprise-version/upgrade/upgrade-vmware/airgap.md

```diff
@@ -0,0 +1,186 @@
+
+    ```shell
+    ssh -identity_file </path/to/private/key> <username>@<vm-ip-or-domain>
+    ```
+
+    Consider the following command example for reference.
+
+    ```shell
+    ssh -identity_file /docs/ssh-private-key.pem ubuntu@palette.example.com
+    ```
+
+2.  Use the following command to switch to the `root` user account, which you need to proceed with the upgrade.
+
+    ```shell
+    sudo --login
+    ```
+
+3.  Use the following command template to download the new Palette airgap installation bin. Enter the username,
+    password, and the Palette airgap installation URL you received from our support team. In the output file name,
+    replace `<version>` with the Palette version you're downloading.
+
+    ```shell
+    curl --user <username>:<password> --output airgap-<version>.bin <link-to-airgap-installation-bin>
+    ```
+
+    Consider the following command example for reference.
+
+    ```shell
+    curl --user <username>:<password> --output airgap-4.1.12.bin https://software-private.spectrocloud.com/airgap/4.1.12/airgap-v4.1.12.bin
+    ```
+
+4.  Refer to the [Additional Packs](../../install-palette/airgap/supplemental-packs.md) page and update the packages you
+    are currently using. You must update each package separately.
+
+    To update a package, use the following command template to download and execute the pack binary.
+
+    ```shell
+    curl --remote-name <pack-name-url> && chmod +x <pack-name-version>.bin && ./<pack-name-version>.bin
+    ```
+
+    Consider the following example for reference.
+
+    ```shell
+    curl --remote-name https://software-private.spectrocloud.com/airgap/packs/airgap-pack-aws-alb-2.5.1.bin \
+      && chmod +x airgap-pack-aws-alb-2.5.1.bin \
+      && ./airgap-pack-aws-alb-2.5.1.bin
+    ```
+
+    ```shell
+    Verifying archive integrity...  100%   MD5 checksums are OK. All good.
+    Uncompressing Airgap Pack - aws-alb Version 4.0.17  100%
+    Setting up Packs
+    - Pushing Pack aws-alb:2.5.1
+    Setting up Images
+    Setup Completed
+    ```
+
+5.  Use the following command template to execute the new Palette airgap installation bin.
+
+    ```shell
+    chmod +x airgap-<version>.bin && ./airgap-<version>.bin
+    ```
+
+    Consider the following example for reference.
+
+    ```shell
+    chmod +x airgap-v4.2.13.bin && ./airgap-v4.2.13.bin
+    ```
+
+6.  After the Palette airgap binary is verified and uncompressed, it uploads the updates to your existing Palette
+    instance. This step takes some time to complete.
+
+    Once the airgap binary is executed, you will receive a **Setup Completed** success message.
+
+    ```shell
+    Verifying archive integrity...  100%   MD5 checksums are OK. All good.
+    Uncompressing Airgap Setup - Version 4.2.13  100%
+    Setting up CLI
+    Setting up Manifests
+    Setting up Packs
+    ...
+
+    //highlight-next-line
+    Setup Completed
+    ```
```

Comment:
Moving the output example before the bullet makes it easier for the reader, as they have a better mental context.